webpack/base.js

```diff
@@ -5,11 +5,11 @@ module.exports = () => {
   return {
     entry: {
       "login": [
-        "./src/apps/loyalty/containers/login/browser",
+        "babel-polyfill", "./src/apps/loyalty/containers/login/browser",
       ],
       "inquiries": [
-        "./src/apps/loyalty/containers/inquiries/browser",
-      ]
+        "babel-polyfill", "./src/apps/loyalty/containers/inquiries/browser",
```

Comment:
Oh my bad, I now see there were 2 commits indeed. Did you by any chance verify if webpack is tree shaking most of the polyfill out? (i.e. only including required code in the bundles)